blocks/init/components/carousel-navigation/carousel-navigation.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+
+export const CarouselNavigation = () => {
+
+  const componentClass = 'carousel-navigation';
+
+  const arrowItemPrevClass = `
+    ${componentClass}__arrow-item
+    ${componentClass}__arrow-item--prev
+  `;
+
+  const arrowItemNextClass = `
+    ${componentClass}__arrow-item
+    ${componentClass}__arrow-item--next
+  `;
+
+  const arrowIconClass = `
+    ${componentClass}__arrow-icon
+  `;
+
+  return (
+    <div className={componentClass}>
+      <div className={arrowItemPrevClass}>
+        <svg className={arrowIconClass} xmlns="http://www.w3.org/2000/svg" width="30" height="12">
+          <path fill="#353535" fillRule="evenodd" d="M7.174 6v4.483L.686 5.657 7.174.83V5H30v1H7.174z" />
+        </svg>
+      </div>
+      <div className={arrowItemNextClass}>
+        <svg className={arrowIconClass} xmlns="http://www.w3.org/2000/svg" width="30" height="12">
+          <path fill="#353535" fillRule="evenodd" d="M22.826 5V.83l6.488 4.827-6.488 4.826V6H0V5h22.826z" />
```

Comment:
```suggestion
          <path fill="currentColor" fillRule="evenodd" d="M22.826 5V.83l6.488 4.827-6.488 4.826V6H0V5h22.826z" />
```

Reply:
I'm not sure if this is possible to implement since it depends on implementation inside this carousel but to me it seems as a good suggestion. @iruzevic what do you think?

Reply:
It makes sense, but this is just a simple component you would add your project implementation to this. In 99% you will not have this kind of arrow. So this improvement is ok but will think about this. thx